docs/src/guides/run-genbank-ingest-on-terra.rst

```diff
@@ -0,0 +1,65 @@
+*********************************************
+Ingest SARS-CoV-2 data from GenBank on Terra
+*********************************************
+
+We have provided two pipelines for importing data into Terra:
+
+* **GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.**
+* GISAID Ingest - pull a private dataset if a user has their own API endpoint, username, and password.
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagrammed below:
+
+.. image:: ../images/terra-ingest.png
+
+This guide describes the **GenBank Ingest**.
+
+Import the GenBank ingest wdl workflow from Dockstore
+======================================================
+
+1. `Set up a Terra account <https://terra.bio/>`_.
+2. Navigate to one of the following in Dockstore:
+    - `nextstrain/ncov/genbank_ingest`_: for open (GenBank) data
+3. At the top right corner, under **Launch with**, click on **Terra**. You may be prompted to log in.
+4. Provide a **Workflow Name** (e.g. ``genbank_ingest`` ).
+5. Select a **Destination Workspace** from the dropdown menu.
+6. Click **IMPORT**.
+7. In your workspace, click on the **WORKFLOWS** tab and verify that the imported workflow is showing a card.
+
+.. _`nextstrain/ncov/genbank_ingest`: https://dockstore.org/workflows/github.com/nextstrain/ncov/genbank_ingest:master?tab=info
+
+Connect any workspace variables to the wdl ingest workflow
+===========================================================
```

Comment:
The steps in this section work, which is great! But, as someone who is unfamiliar with Terra, I don't really understand what I'm doing here. Some lingering questions:

1. What does "root entity type" mean, and what is `ncov_examples`?
2. > "Select the 1st row in the data table. The first column should have value `blank`". Selecting more rows will cause the workflow to run more than once.
    1. This works, but seems hacky. What does `blank` mean?
    2. Why are there other rows available if we aren't selecting them? Are those for other use cases?
3. For the **OUTPUTS** tab:
    1. Why can't I just use the default values `this.*`?
    2. What's the significance of using `workspace.*`?
    3. What about the `last_run` variable?